test/scripts/setup-dep.sh

```diff
@@ -21,7 +54,7 @@ function check_kustomize() {
     echo "This script will install kustomize on your machine"
     curl -LO "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
     chmod +x ./install_kustomize.sh
-    source ./install_kustomize.sh 3.8.2 ${bin_dir}
+    sudo ./install_kustomize.sh 4.5.7 ${bin_dir}
```

Comment:
`source` or `sudo`? `sudo` is not correct.

Reply:
`sudo` since we need the higher privilege to move the binary to `bin_dir`

Reply:
Do not need to `source` or `.`?